includes/container/container.php

```diff
@@ -4,6 +4,7 @@
 
 use ElementorDeps\DI\ContainerBuilder;
 use ElementorDeps\DI\Container as DIContainer;
+use Exception;
```

Comment:
@Batsirai-muchareva If the class is under global NS you don't need to "use" it. Just call by `\Exception`.